parm/archive/gdas.yaml.j2

```
@@ -167,6 +171,11 @@ gdas:
         - "{{ COMIN_ATMOS_HISTORY | relpath(ROTDIR) }}/{{ head }}atm.f{{ fhr3 }}.nc"
         - "{{ COMIN_ATMOS_HISTORY | relpath(ROTDIR) }}/{{ head }}sfc.f{{ fhr3 }}.nc"
         {% endfor %}
+
+
+        # Forecast Master GRIB2 data for F006
+        - "{{ COMIN_ATMOS_MASTER | relpath(ROTDIR) }}/{{ head }}master.f003.grib2"
+        - "{{ COMIN_ATMOS_MASTER | relpath(ROTDIR) }}/{{ head }}master.f006.grib2"
```

Should we include the `idx` files?

---

Actually, I wonder if we are even generating the index files for these.  Probably are not.

---

I had thought about this as well.  We're at least not saving them to COM, we don't have them in our parallels.  I didn't check if there is a silent failure somewhere though.

@WenMeng-NOAA - Can you confirm whether the master grib files should have idx files saved in COM as well?

---

The `idx` files for the master are not created, similar to the `sflux` files.  The `sflux` `idx` files are generated in the `atmos_products` task.  I'll start generating the `master.idx` files if the group wants that, and start including it in `COM`.  The `master` files are strictly internal to GFS (until now for archiving for AIGFS).